inconspiquous/dialects/gate.py

```diff
@@ -120,6 +120,15 @@ def __init__(self, gate: GateAttr):
 class HadamardGate(SingleQubitGate):
```

Comment:
This should be a `CliffordGateAttr`, and will also need a `num_qubits` property (alternatively you could make `SingleQubitCliffordGate` and `TwoQubitCliffordGate` subclasses of `CliffordGateAttr`